edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/handlers/ConnectionHandler.cs

```diff
@@ -94,6 +99,20 @@ public async Task<bool> HandleAsync(MqttPublishInfo publishInfo)
             return false;
         }
 
+        public async Task CloseConnectionAsync(IIdentity identity)
+        {
+            var identityFound = false;
+            using (await this.guard.LockAsync())
+            {
+                identityFound = this.knownConnections.TryGetValue(identity, out var _);
+            }
+
+            if (identityFound)
+            {
+                await this.connector.SendAsync(TopicDisconnect, Encoding.UTF8.GetBytes($"\"{identity.Id}\""));
```

Comment:
do we need to remove from knownConnections?

Reply:
No, I figured that after this point it is the turn of the broker to do something, and when it make the client disconnect, the notification will come in a second round, and the identity will be removed that time